Sumangala1.c

```diff
@@ -0,0 +1,11 @@
+//program to find the sum inclusive of the start and the end no. mentioned
```

Comment:
Very Good. Appreciate your comment on what the program is doing.
Also leave a comment block about the file created date and author something like this : 

/*
Created on October 22, 2018
@author: anoopbhatn
*/